internal/cli/atlas/privateendpoints/gcp/interfaces/describe.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package interfaces
+
+import (
+	"context"
+
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/cli/require"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type DescribeOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	privateEndpointServiceID string
+	privateEndpointGroupID   string
+	store                    store.InterfaceEndpointDescriber
+}
+
+func (opts *DescribeOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+var describeTemplate = `ENDPOINT	STATUS	DELETE REQUESTED
+{{.EndpointGroupName}}	{{.Status}}	{{.DeleteRequested}}
+`
+
+func (opts *DescribeOpts) Run() error {
+	r, err := opts.store.InterfaceEndpoint(opts.ConfigProjectID(), provider, opts.privateEndpointServiceID, opts.privateEndpointGroupID)
+
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+// mongocli atlas privateEndpoint(s) gcp interface(s) describe <endpointGroupId> --endpointServiceId endpointServiceId [--projectId projectId].
+func DescribeBuilder() *cobra.Command {
+	opts := new(DescribeOpts)
+	cmd := &cobra.Command{
+		Use:     "describe <endpointGroupId>",
+		Aliases: []string{"get"},
+		Args:    require.ExactArgs(1),
+		Short:   "Return a specific GCP private endpoint interface for your project.",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			opts.privateEndpointGroupID = args[0]
```

Comment:
you need to add `Annotations` and `Example`. See this command for an example https://github.com/mongodb/mongocli/blob/8247c6f0a5d45ecba016a79cf73459eecadeb227/internal/cli/atlas/privateendpoints/gcp/delete.go#L59

Reply:
Thanks, I've added those now.